.travis.yml

```diff
@@ -1,22 +0,0 @@
-language: ruby
-gemfile:
-  - gemfiles/rails5_0.gemfile
-  - gemfiles/rails5_1.gemfile
-  - gemfiles/rails5_2.gemfile
-  - gemfiles/rails6_0.gemfile
-rvm:
-  - jruby-19mode
```

Comment:
It seems to be using JRuby 1.7.9

* https://github.com/rvm/rvm/blob/master/patchsets/jruby/1.7.9/default
* https://github.com/jruby/jruby/releases/tag/1.7.9

Since it's really old; I don't think we should stick with 1.7.9 in my opinion